### PR TITLE
Fix sentry in worker

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Configure sentry only for the first time.
- Fixes RED-HAT-0P-2AV (I hope.)

### CODE REVIEW

File: tests/unit/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
```
Comment (csomh) [inline tests/unit/test_utils.py]: 2020?

Comment (lachmanfrantisek) [reply tests/unit/test_utils.py]: Please, create a new (good-first?) issue -- it's in all files..;(

**Comment by lachmanfrantisek** [comment]
>     1. `lazy_object_proxy` should become a dependency and installed in `files/install-deps*.yaml`. Maybe it's already indirectly pulled in by something else, still, might be better to be explicit.

It should be already there because of the `packit`. I'll check it once again.

>     2\. I'm not really familiar with lazy initialisation. Is there be any considerable performance impact with this?

Nothing I am aware of -- it only postpones the creation of an object until you need to use it. (No big magic.)

**Comment by lachmanfrantisek** [comment]
@csomh is the `setup.cfg` in packit enough?

https://github.com/packit-service/packit/blob/5fcddebb447a465e2c6deb64d9b5fd643fac2739/setup.cfg#L42

**Comment by csomh** [comment]
> @csomh is the `setup.cfg` in packit enough?
> 
> https://github.com/packit-service/packit/blob/5fcddebb447a465e2c6deb64d9b5fd643fac2739/setup.cfg#L42

As this has become an explicit dependency for `packit-service` would be better to have it here, too. So that things don't break here, if it gets removed from `packit`.

**Comment by lachmanfrantisek** [comment]
> As this has become an explicit dependency for `packit-service` would be better to have it here, too. So that things don't break here, if it gets removed from `packit`.

This is applicable to multiple packit dependencies -- do we really want all of them explicitly marked here as well? (The tests will tell us if we break that, won't they?)

**Comment by csomh** [comment]
> > As this has become an explicit dependency for `packit-service` would be better to have it here, too. So that things don't break here, if it gets removed from `packit`.
> 
> This is applicable to multiple packit dependencies -- do we really want all of them explicitly marked here as well? (The tests will tell us if we break that, won't they?)

packit-service and packit are two separate projects. Implicitly inheriting dependencies from a different project is bad, as it creates a tight coupling between the two.

Are we sure that the coverage of our tests is good enough to catch missing dependencies?

And yes, it's clear that many dependencies of `packit-service` are not explicitly stated as such. That can be fixed as part of a different effort. In this PR we should just make sure not to add yet another item to that list.

**Comment by lachmanfrantisek** [comment]
> Are we sure that the coverage of our tests is good enough to catch missing dependencies?

If we import each file of the code in tests, then yes. (I think we do that.)

(I've added the dependency.)

**Pull request merged**